### PR TITLE
fix: MD042/no-empty-links

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Comment by PRMerger7** [comment]
@nschonni : Thanks for your contribution! The author, @TerryGLee, has been notified to review your proposed change.

File: docs/vs-2015/install/create-an-offline-installation-of-visual-studio.md [hunk 1]
```diff
@@ -81,4 +81,4 @@ This page describes how to install Visual Studio 2015 when you are not connected
 |        Turkish        | http://go.microsoft.com/fwlink/?LinkID=785882&clcid=0x41F |
 
 ## See Also
- [Install Visual Studio]()
\ No newline at end of file
+ [Install Visual Studio](../install/install-visual-studio.md)
```
Comment (TerryGLee) [inline docs/vs-2015/install/create-an-offline-installation-of-visual-studio.md]: The page for which you've suggested an edit is in our Visual Studio 2015 library, so the link must be "../install/install-visual-studio-2015.md"

Comment (TerryGLee) [reply docs/vs-2015/install/create-an-offline-installation-of-visual-studio.md]: Please fix per my previous comment; thanks. :)

Comment (nschonni) [reply docs/vs-2015/install/create-an-offline-installation-of-visual-studio.md]: Fixed! I noticed when I mad the change that it was a redirect file, but i figured that might have been fine.

**Comment by TerryGLee** [comment]
#sign-off

**Pull request merged**